index.js

```diff
@@ -15,9 +16,12 @@ let test = regex => str => regex.test(str) // This mirrors the test function in
 
 // Core
 let defaultFilter = _.filter(test(/.js|.html|.jsx|.ts|.coffee|.less|.css|.sass|.hbs|.ejs/))
-let metagen = dir => relativeFilenames(dir.path, dir.exclusions || [dir.output || '__all.js'])
-  .then(dir.filter || defaultFilter)
-  .then(files => fs.writeFileAsync(dir.path + (dir.output || '__all.js'), dir.format(files, dir)))
+let metagen = options => relativeFilenames(options.path, options.exclusions || [options.output || '__all.js'])
+  .then(options.filter || defaultFilter)
+  .then(files => fs.writeFileAsync(
+      path.join(options.path, (options.output || '__all.js')),
+      options.format(files.map(file => file.replace(/^\//g, '')), options))
```

Comment:
we can use lodash/fp's `replace` so we don't need to create a function inside map and leverage the curry